public/app/core/routes/dashboard_loaders.js

```diff
@@ -7,11 +7,25 @@ function (coreModule) {
   coreModule.default.controller('LoadDashboardCtrl', function($scope, $routeParams, dashboardLoaderSrv, backendSrv) {
 
     if (!$routeParams.slug) {
-      backendSrv.get('/api/dashboards/home').then(function(result) {
-        var meta = result.meta;
-        meta.canSave = meta.canShare = meta.canStar = false;
-        $scope.initDashboard(result, $scope);
+
+      backendSrv.get('/api/preferences').then(function(preferences) {
```

Comment:
there is a backend api route to get home dashboard, '/api/dashboards/home` , think all of this should be handled in that